src/Shared/HandyControl_Shared/Controls/Other/ScrollViewer.cs

```diff
@@ -17,6 +17,8 @@ public class ScrollViewer : System.Windows.Controls.ScrollViewer
 
         private bool _isRunning;
 
+        private Storyboard _scrollingAnimation = new Storyboard();
```

Comment:
@M0n7y5 emmm, can you tell me the use of _scrollingAnimation? Or why commented BeginAnimation on line 123 but not on line 141?

Reply:
Oh ... That's a my mistake ... Sorry a forgot to commit my recent code, so this pull request contains old mess 👀 Can i just close this pull request and make new ?

Reply:
Just let me do it.

Reply:
k